.changes/v3.9.0/1014-notes.md

```diff
@@ -0,0 +1,3 @@
+* Removed disk update steps from `TestAccVcdIndependentDiskBasic`, created a new one `TestAccVcdIndependentDiskBasicWithUpdates` which will be run only on new releases of VCD (>=v10.4.1) [GH-1014]
```

Comment:
Please add the "why" this has been done too, so the motivation is instantly clear for the reader.

And a small ask: please update PR's name to contain a word or two about which "flaky tests" it is fixing (i.e. independent disk).

Reply:
Updated the entry with some additional context, not sure if I should give more context or if this is enough in this case.